install/configuring_inventory_file.adoc

```diff
@@ -2018,3 +2018,58 @@ following variables can be used to customize the cluster console.
 |Optional CA file to use to connect to the OAuth server. Defaults to `/var/run/secrets/kubernetes.io/serviceaccount/ca.crt`. Typically this does not need to be changed.
 
 |===
+
+[[configuring-the-operator-lifecycle-manager]]
+== Configuring the Operator Lifecycle Manager
+
+ifdef::openshift-enterprise[]
+[IMPORTANT]
+====
+The Operator Framework is a Technology Preview feature.
+Technology Preview features are not supported with Red Hat production service
+level agreements (SLAs), might not be functionally complete, and Red Hat does
+not recommend to use them for production. These features provide early access to
+upcoming product features, enabling customers to test functionality and provide
+feedback during the development process.
+
+For more information on Red Hat Technology Preview features support scope, see
+https://access.redhat.com/support/offerings/techpreview/.
+====
+endif::[]
+
+Installing the Technology Preview
+link:https://coreos.com/blog/introducing-operator-framework[Operator Framework]
+is handled via an Operator. You can install the Operator Lifecycle Manager (OLM)
```

Comment:
Is the installing of the OLM handled by the Operator? @ecordell

Reply:
I wondered about this line as well; came from https://docs.google.com/document/d/1Rh3bSMw0CPnZQBws5AnhBBcesL-Mcs8b9xvga-icdG4/edit#heading=h.2abu0i1tlsjd. Can drop it.

Reply:
I've dropped it.